components/brave_wallet/browser/cardano/cardano_get_utxos_task.cc

```diff
@@ -51,27 +48,23 @@ void GetCardanoUtxosTask::MaybeSendRequests() {
     return;
   }
 
-  // Shuffle addresses so requests are always done in different order to
-  // increase privacy a bit.
-  base::RandomShuffle(addresses_.begin(), addresses_.end());
```

Comment:
why do we drop shuffling?

Reply:
Cardano is less oriented towards privacy. There would be less addresses in account in general and each address within account has same stake credentials part.